src/main/io/osd.h

```diff
@@ -474,6 +490,10 @@ typedef struct osdConfig_s {
     uint16_t adsb_ignore_plane_above_me_limit;          // in metres
 #endif
     uint8_t  radar_peers_display_time;                  // in seconds
+#ifdef USE_GEOZONE
+    uint8_t geozoneDistanceWarning;                     // Distance to fence or action
+    bool geozoneDistanceType;                            // Shows a countdown timer or distance to fence/action
+#endif
 } osdConfig_t;
 
 PG_DECLARE(osdConfig_t, osdConfig);
```

Comment:
You need to bump the version number of osdConfig_t in osd.c

Look for a line like:
```
PG_REGISTER_WITH_RESET_TEMPLATE(osdConfig_t, osdConfig, PG_OSD_CONFIG, 13);
```